filebeat/docs/autodiscover-hints.asciidoc

```diff
@@ -38,8 +37,18 @@ include_lines
 
 [float]
 ==== Docker
-Docker allows providing hints through Docker labels. When containers are spun up the Builder understands
-labels that convey hints to convey additional configuration to the input.
+
+Docker autodiscover provider supports hints too, it's enabled in the same way:
```

Comment:
I would not use `too` here as people might jump to this section of the docs directly.